x-pack/test/security_solution_api_integration/es_archive_path_builder/index.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { SERVERLESS_ES_ARCHIVE_PATH, ESS_ES_ARCHIVE_PATH } from './constants';
+
+export class EsArchivePathBuilder {
```

Comment:
nit: Is there a plan to extend this class later on? It looks like overkill for now as it could be just a helper function like `getEsArchivePath(type: 'ess' | 'serverless', resourceUri: string): string`.

Reply:
This class came as Devin, brought up a great point here https://github.com/elastic/kibana/pull/166755#discussion_r1343091446